web_list_html_widget/README.rst

```diff
@@ -0,0 +1,45 @@
+Widget Html rendering in list views
+===================================
+
+This module allows to use a ``widget="html"`` for fields in list(tree) views.
+
+*WARNING* The content of the field will be rendered without HTML escaping and without any other security measure.
```

Comment:
Actually, I didn't succeeded in breaking Odoo security with this widget. Maybe someone can confirm/invalidate this ?
FYI, I made some tests from this site : https://www.owasp.org/index.php/Testing_for_Cross_site_scripting